webapp/channels/src/components/post_edit_history/edited_post_item/edited_post_items.scss

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.edit-post-history__container {
+    .edit-post-history__icon__button.toggleCollapseButton {
+        pointer-events: none;
```

Comment:
Why don't we want this to receive pointer events? If this isn't clickable, it shouldn't be a button to begin with.

Reply:
It is clickable. The thing is, the event handler to toggle the edit history item is applied on the whole edit history item. This button was working in a reverse way - blocking click events from reaching the underlying component and thus, preventing you from toggling the edit history item when clicking on this button. Removing pointer events from this continues the button to work and allow toggling the edit history item, albeit in a hacky way.

Reply:
Ah, I see. That makes sense in this case even though I feel like the underlying UX is what's making us need to include this weirder logic. I feel like we run into this a lot where we have behaviour that makes a very large area a button (like how clicking anywhere on a post can open the RHS)